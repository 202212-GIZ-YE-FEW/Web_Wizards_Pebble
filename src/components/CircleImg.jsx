const CircleImg = () => {
    return (
        <div className='w-2/12 '>
            <div className='flex items-center justify-center w-32 h-32 md:w-36 md:h-36 lg:w-64 lg:h-64 rounded-full bg-black-100 text-white text-3xl font-bold'>
                <span className=''>R</span>
            </div>
        </div>
    );
};

export default CircleImg;
