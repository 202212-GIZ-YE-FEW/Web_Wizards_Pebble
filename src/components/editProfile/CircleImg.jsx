import { useAuthContext } from "@/context/AuthContext";

const CircleImg = () => {
    const { user } = useAuthContext();
    const firstLetter = user?.displayName.at(0);
    return (
        <div className='flex items-center justify-center w-32 h-32 md:w-36 md:h-36 lg:w-64 lg:h-64 rounded-full bg-black-100 text-white text-3xl font-bold'>
            <span className=''>{firstLetter}</span>
        </div>
    );
};

export default CircleImg;
