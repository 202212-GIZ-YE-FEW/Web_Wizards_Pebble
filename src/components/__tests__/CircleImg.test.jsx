import renderer from "react-test-renderer";

import CircleImg from "@/components/CircleImg";

it("renders correctly", () => {
    const tree = renderer.create(<CircleImg />).toJSON();
    expect(tree).toMatchSnapshot();
});
