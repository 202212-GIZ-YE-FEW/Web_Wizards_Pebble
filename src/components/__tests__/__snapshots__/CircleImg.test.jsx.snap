// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="flex items-center justify-center w-32 h-32 md:w-36 md:h-36 lg:w-64 lg:h-64 rounded-full bg-black-100 text-white text-3xl font-bold"
>
  <span
    className=""
  />
</div>
`;
